src/js/wall/FirstUser.jsx

```diff
@@ -12,9 +11,23 @@ const initialValues = {
     password: "",
 };
 
-export const FirstUser = ({ onSubmit, errors }) => {
+export default function FirstUser() {
+    const [error, setError] = useState("");
```

Comment:
Why not use the error handling baked into useMutation? https://tanstack.com/query/v4/docs/react/guides/mutations